charts/fluent-bit/fluent-bit-splunk/v1.7.8/values.yaml

```diff
@@ -0,0 +1,268 @@
+# Default values for fluent-bit.
+
+# kind -- DaemonSet or Deployment
+kind: DaemonSet
+
+# replicaCount -- Only applicable if kind=Deployment
+replicaCount: 1
+
+image:
+  repository: fluent/fluent-bit
+  pullPolicy: Always
```

Comment:
Is this _Always_ by default? Any reason for this?

Reply:
Yes, the image is fetched from https://hub.docker.com/r/fluent/fluent-bit/